packages/container/README.md

```diff
@@ -0,0 +1,40 @@
+# @kaizen/container
+
+## About
+Container combines all the boilerplate and setup to ensure Kaizen components are consistent across different codebases. This includes:
+* Theme management
+* Browser normalisation 
+* Fonts
+* CSS Polyfills
+
+## Install
+```
+yarn add @kaizen/container
+```
+
+### Usage
+To use, wrap App in the React component: 
+
+```
+// App.tsx
+import { Container } from "@kaizen/container"; 
+
+const YourApp = () => (
+  <Container>
+    /* your React app goes here */
+  </Container>
+);
+```
+
```

Comment:
I had a thought that we should document usage on non-react pages:

````suggestion
### Usage
To use, wrap App in the React component: 

```
// App.tsx
import { Container } from "@kaizen/container"; 

const YourApp = () => (
  <Container>
    /* your React app goes here */
  </Container>
);
```

You can also use this on non-react pages to initialise the theme manager and get the required font definitions, polyfills etc:

```
import "@kaizen/container"; 
```
````